api/v1alpha1/groupversion_info.go

```diff
@@ -14,9 +14,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package v1alpha1 contains API Schema definitions for the odf v1alpha1 API group
-//+kubebuilder:object:generate=true
-//+groupName=odf.openshift.io
+// Package v1alpha1 contains API Schema definitions for the ocs v1alpha1 API group
```

Comment:
Any specific reason for changing the group name? This is *technically* part of ODF.